<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_ImageOptions</class>
 <widget class="QWidget" name="Ui_ImageOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>22</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Insert Options</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QHBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="lAngle">
     <property name="text">
      <string>Angle:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leAngle">
     <property name="toolTip">
      <string>Rotation Angle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lFactor">
     <property name="text">
      <string>Scale Factor:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leFactor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Scale Factor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lDPI">
     <property name="text">
      <string>DPI:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leDPI">
     <property name="toolTip">
      <string>DPI of Image</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>updateFactor()</slot>
 </slots>
</ui>
